service/docs/source/requires.rst

```diff
@@ -44,123 +39,99 @@ Upstream SLES and OpenSUSE Package Requirements
-access managed through an allowed list that is controlled by the system
-administrator.  This feature is required by the GEOPM runtime if the
-GEOPM Service is not active on the system.  Alternately, the access
-management for MSRs may be configured by the system administrator
-using the GEOPM Service if it is active.
-
-The msr-safe kernel driver is distributed with OpenHPC and can be
-installed using the RPMs distributed there.  The source code for the
-driver can be found `here <https://github.com/LLNL/msr-safe>`__.
-
-If both the msr-safe kernel driver and the GEOPM Systemd Service are
-unavailable, then GEOPM when run by the root user may access MSRs
-through the standard msr driver.  This may be loaded with the
-following command:
+The msr-safe kernel driver provides two key features. Firstly, it offers a low
+latency interface for reading and writing many MSR values at once through an
+`ioctl(2) <https://man7.org/linux/man-pages/man2/ioctl.2.html>`_ system call, possibly
+improving the performance of GEOPM HPC runtime or other MSR usages.
+
+Secondly, the msr-safe kernel driver enables user-level read and write operations of
+the model-specific registers (MSRs) with access controlled by the system administrator.
+This feature is mandatory if the GEOPM Service is not active on the system. Alternatively, the access can
+also be managed by the system administrator using the GEOPM Service, if active.
+
+The msr-safe kernel driver code can be found `here <https://github.com/LLNL/msr-safe>`__.
+It's distributed with OpenHPC and can be installed from the RPMs provided there.
+
+In the absence of both the msr-safe kernel driver and the GEOPM Systemd Service, root users may access MSRs
+using the standard MSR driver. This can be loaded with the command:
 
 .. code-block:: bash
 
     modprobe msr
 
-The standard msr driver must also be loaded to enable MSR access
+The standard MSR driver must also be loaded to enable MSR access
 through the GEOPM Systemd Service when msr-safe is not installed.
 
-
 Systemd Configuration
 ^^^^^^^^^^^^^^^^^^^^^
 
-In order for GEOPM to properly use shared memory to communicate
-between the Controller and the application, it may be necessary to
-alter the configuration for systemd.  The default behavior of systemd
-is to clean-up all inter-process communication for non-system users.
-This causes issues with GEOPM's initialization routines for shared
-memory.  This can be disabled by ensuring that ``RemoveIPC=no`` is set
-in ``/etc/systemd/logind.conf``.  Most Linux distributions change the
-default setting to disable this behavior.  More information can be
-found `here <https://superuser.com/a/1179962>`__.
+For GEOPM to utilize shared memory for communication between the Controller and the
+application, it may be necessary to alter systemd configuration. By default, systemd
+removes all inter-process communication for non-system users. This poses a problem to
+GEOPM's shared memory initialization routines.
```

Comment:
This is not an issue any more.  The interprocess shared memory keys are managed in /run/geopm now.